src/core/analytics.js

```diff
@@ -1257,6 +1328,15 @@ class Analytics {
   endSession() {
     this.uSession.end();
   }
+
+  setAuthToken(token) {
```

Comment:
can we be more explicit with naming here like setDMTAuthToken or something to avoid confusion with our basic authendication header token

Reply:
I can do that, Android and iOS implementations also have the same kind of name. 
Actually, we never decided on the public-facing API name for this.

Reply:
agreed